src/plugins/content/accessibility/keyNavigation/modes/nativeMode.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 Open Assessment Technologies SA ;
+ */
+
+/**
+ * Native key navigation mode
+ */
+export default {
+    name: 'native',
+
+    /**
+     * Builds the key navigation config for the "native" mode
+     * @param {keyNavigationStrategyConfig} config - additional config to set
+     * @returns {keyNavigationMode}
+     */
+    init(config = {}) {
+        return {
+            // todo: add access to the page and the rubric blocks
+            strategies: ['header', 'navigator', 'item', 'toolbar'],
```

Comment:
when I run `test:keepAlive` for `test/plugins/content/keyNavigation/plugin/test.html` in mode `Native`  I didn't reach left navigation panel by the keyboard. 
Or strategy `'navigator' is not left navigation panel?

Reply:
The first item of the mock test is in a linear section, so the navigator is not interactive. If you move to the next item, you should be able to reach it. However, the visual style may not be obvious (I didn't change this).